packages/babel-plugin-transform-spread/src/index.js

```diff
@@ -62,12 +62,26 @@ export default declare((api, options) => {
           return;
         }
 
-        path.replaceWith(
-          t.callExpression(
-            t.memberExpression(first, t.identifier("concat")),
-            nodes,
-          ),
-        );
+        this.addHelper("toConsumableArray");
+
+        const toConsumableArrayExpression = obj =>
+          t.callExpression(t.identifier("_toConsumableArray"), obj);
```

Comment:
I don't have time to think about this PR as a whole at the moment, but FYI this identifier is returned from `addHelper`, you shouldn't create it manually.